ntbs-service/Models/Validations/ValidationHelper.cs

```diff
@@ -36,7 +36,9 @@ public static string ValidYearLaterThanBirthYear(string name, int birthYear)
         #endregion
 
         #region Patient Details
-        public const string NhsNumberLength = "{0} needs to be 10 digits long";
+        public const string NhsNumberLength = "NHS number needs to be 10 digits long";
+        public const string InvalidNhsNumber = "This NHS number is not valid. Confirm you have entered it correctly";
+        public const string NhsNumberFormat = "NHS number can only contain digits 0-9";
```

Comment:
I think we could still use the `{0}` formatting here, so that we benefit from display name being defined in just one place